cli/script/command-executor.ts

```diff
@@ -1223,6 +1364,43 @@ function requestAccessToken(): Promise<string> {
     });
 }
 
+export var runReactNativeBundleCommand = (entryFile: string, outputFolder: string, platform: string, sourcemapOutput: string): Promise<void> => {
+    var reactNativeBundleCommandArgs = [
```

Comment:
On a quick glance... using local-cli seems possible. I'll need to investigate it further, but all the global-cli seems to do in the case of the `react-native bundle` command is try to find the path to the local CLI and require it. It does a lot more e.g includes a lot of troubleshooting information for the user in the case the user is running "react-native init", but that's not what we care about.